kamailio/fast-pickup-role.cfg

```diff
@@ -153,7 +153,7 @@ route[FAST_PICKUP_INIT]
    $var(Option) = $(var(Pickup){s.encode.hexa});
    $var(Cookie) = $(kzE{kz.json,Switch-URI}{s.md5});
    $var(call_id) = $(kzE{kz.json,Call-ID});
-   $var(JObj) = $(kzE{re.subst,/"Call-ID"\s*\:\s*"([^\s"]*)"/"Call-ID" : "kfp+$var(Option)$var(Cookie)@\1"/});
+   $var(JObj) = $(kzE{re.subst,/"Call-ID"\s*\:\s*"([^ "]*)"/"Call-ID" : "kfp+$var(Option)$var(Cookie)@\1"/});
```

Comment:
did you tried `([^\\s"]*)` ?

Reply:
yeah I tried \\s and \\\\s (in case it was needing to be double-escaped). Neither worked

Reply:
@danielfinke i'm not a fan of this solution (space should be properly coded) because re can ignore whitespaces , but it seems to me that this would work without the space, `([^"]*)` (any char different than terminating `"`). can you try this ?

Reply:
@lazedo that works for me. I also don't see a reason why there would be whitespace in the call ID